src/config/dnsmasq_config.c

```diff
@@ -547,6 +552,14 @@ bool __attribute__((nonnull(1,3))) write_dnsmasq_config(struct config *conf, boo
 		fputs("local=//\n\n", pihole_conf);
 	}
 
+	// Ensure that home.arpa domains (RFC 8375) are not sent upstream, unless configured by
+	// user as local domain, with server setting applied above
+	if (domain_homearpa == false)
+	{
+		fputs("# Do not forward home.arpa domains to upstream servers\n",pihole_conf);
+		fputs("server=/home.arpa/\n\n",pihole_conf);
```

Comment:
```suggestion
		fputs("local=/home.arpa/\n\n",pihole_conf);
```
I know, nit-picking here, `local` is a synonym for `server` to make configuration files clearer in this case.